conf/deployment/vsphere/upi_1az_rhcos_vmfs_mcg_only_3m_3w.yaml

```diff
@@ -0,0 +1,18 @@
+---
+# This config is suppose to work on most of DCs we have.
+DEPLOYMENT:
+  allow_lower_instance_requirements: false
+ENV_DATA:
+  platform: 'vsphere'
+  deployment_type: 'upi'
+  mcg_only_deployment: true
+  worker_replicas: 3
+  master_replicas: 3
+  worker_num_cpus: '16'
+  master_num_cpus: '4'
+  master_memory: '16384'
+  compute_memory: '65536'
```

Comment:
I think we can lower memory, please check from docs or devs and decrease the memory + let some free for workloads.